src/components/OrderedList.tsx

```diff
@@ -2,8 +2,13 @@
 import React from "react"
 import styled from "styled-components"
 
-//Styles
-const Content = styled.div`
+export interface IProps {
+  listData: (string | HTMLParagraphElement)[]
+  className?: string
+}
+
+// Styles
+const Content = styled.div<{ size?: `${string}px` }>`
```

Comment:
Feel free to remove the `size` logic from here since we are not using it.

But if you want to leave it, lets define it as a `number` and handle the `px` suffix where we use it.
```suggestion
const Content = styled.div<{ size?: number }>`
```

Reply:
since we are not using the `size` logic, I will remove it to prevent confusion in legacy code. LMK if this is ok! Or I can just update according to ^ suggestion